Descent3/gamesave.cpp

```diff
@@ -395,11 +397,10 @@ void SaveGameDialog() {
   //	ddio_MakePath(pathname, savegame_dir, "*.sav", NULL); -unused
 
   // create savegame directory if it didn't exist before.
-  if (!ddio_DirExists(savegame_dir)) {
-    if (!ddio_CreateDir(savegame_dir)) {
-      DoMessageBox(TXT_ERROR, TXT_ERRCREATEDIR, MSGBOX_OK);
-      return;
-    }
+  std::error_code ec;
+  if (!std::filesystem::create_directories(savegame_dir, ec)) {
```

Comment:
`ec` unused, use the overload without error code (https://en.cppreference.com/w/cpp/filesystem/create_directory (1))

Reply:
`ec` is unused, but this prevents throwing exceptions.